README.md

```diff
@@ -104,7 +104,7 @@ docker run -d -p 5432:5432 \
   -e POSTGRES_USER=${POSTGRESQL_USER} \
   -e POSTGRES_DB=${POSTGRESQL_DATABASE} \
   -e POSTGRES_PASSWORD=${POSTGRESQL_PASSWORD} \
-  -t bcgov/nr-spar-backend-database:snapshot
+  bcgov/nr-spar-backend-database:snapshot
 
 docker run -t -i --net=host \
```

Comment:
You don't need to set the network to use a host configuration, just expose the ports with -p 5432:5432.

Reply:
I will close this one but this still requires a change on the readme file